src/Silex/Provider/RoutingServiceProvider.php

```diff
@@ -30,13 +29,17 @@
  */
 class RoutingServiceProvider implements ServiceProviderInterface
 {
+    protected const FLAG_USE_LOCALE_LISTENER = 'FLAG_USE_LOCALE_LISTENER';
+
     /**
      * @param \Silex\Application $app
      *
      * @return void
      */
     public function register(Application $app)
     {
+        $app[static::FLAG_USE_LOCALE_LISTENER] = true;
```

Comment:
Can we make those constanst name somehow better? What about `BC_FEATURE_FLAG_LOCALE_LISTENER`?